### DISCUSSION TITLE
Seeking consultant with knowledge in RAGE MP and this gamemode's structure.

### DISCUSSION THREAD

**Discussion by juandiegox21** [OwlGamingCommunity/V#44] [Support]
This gamemode's structure called my attention, I haven't tried running it locally as I'm not interested in the gamemode itself but rather the structure it's using. I'm looking for someone, ideally the contributors of this repo who can offer his services as a consultant since I'm trying to create a roleplay gamemode as a side project. @DanielsGTA @JeremyEspresso @ChaosParadise.

I'm a full time software dev for a few years now with experience in certain technologies. Thank you for reading, and please let me know your hourly rate if interested.

**Comment by deveempty** [comment]
@juandiegox21 devempty discord

**Comment by JeremyEspresso** [comment]
> This gamemode's structure called my attention, I haven't tried running it locally as I'm not interested in the gamemode itself but rather the structure it's using. I'm looking for someone, ideally the contributors of this repo who can offer his services as a consultor since I'm trying to create a roleplay gamemode as a side project. @DanielsGTA @JeremyEspresso @ChaosParadise.
> 
> 
> 
> I'm a full time software dev for a few years now with experience in certain technologies. Thank you for reading, and please let me know your hourly rate if interested. 

Hi @juandiegox21, I am not so much into RAGE:MP development anymore. But feel free to drop your discord here and I'll see if I can help you with any questions you have.

**Reply by juandiegox21** [reply]
Thank you @JeremyEspresso, my discord is peterpan6070
